### ISSUE TITLE
[Adaptive Card iOS - SelectAction.Action.Submit.Overlap.Touch.Json]: 'Submit Action' button is not accessible through the swipe gesture. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8439]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.2" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'SelectAction.Action.Submit.Overlap.Touch.Jsons' and activate it.
7. The Card will open.
8. Navigate to the 'Submit Action' button and observe the issue.

**Actual Result:**
While navigating 'SelectAction.Action.Submit.Overlap.Touch' screen using swipe gesture, 'Submit Action' button is not accessible through the swipe gesture.

**Expected Result:**
'Submit Action' button under 'SelectAction.Action.Submit.Overlap.Touch' should be accessible through the swipe gesture.

**Note:**
1. This issue is not repro through the finger touch.
2. This issue is not repro with Keyboard.

**User Impact:**
It will impact the screen reader user if any element is not accessible through the swipe gesture as user will not be able to perform the desired action.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229072041-ba7e0127-b9ef-4fa1-9af8-1eebdfa7e732.mov

### Card JSON

```JSON
NA
```
